app/controllers/v0/performance_monitorings_controller.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module V0
+  class PerformanceMonitoringsController < ApplicationController
+    def create
+      response = Benchmark::Performance.metrics_for_page(page_id, metrics_data)
+
+      render json: { page_id: page_id, response: response }, serializer: PerformanceMonitoringSerializer
+    end
+
+    private
+
+    def performance_params
+      params.permit(:page_id, metrics: [:metric, :duration])
+    end
+
+    def page_id
+      performance_params['page_id']
+    end
+
+    def metrics_data
+      performance_params['metrics']
```

Comment:
The metrics are coming from the FE in a stringified JSON blob, so this will have to be: 

```rb
JSON.parse(performance_params['metrics'])
```

Reply:
IMO we want the BE endpoint to accept JSON, as is.  Reason being, it follows the same pattern that most all of our other `POST/PUT` BE endpoints use (for consistency), and it helps future FE developers understand the endpoint, through the swaggers docs (as they are accustomed to).  

Meaning we can clearly document and define everything about the endpoint's requirements in the swagger documents.  Providing sample structure, definitions of individual properties in the structure (including examples), illustrating required fields, and more.  

If we change this to expect a stringified JSON blob, our only option for documenting everything about the endpoint would have to be through one big note.

That all said, there should be precedence for the FE passing the JSON in a `POST`.  I say that based on other BE endpoints that receive similar payloads from the FE (i.e. the Vet360 `POST/PUT` endpoints for updating a person's address, email, telephone, etc.).